### ISSUE TITLE
Fastform not accept copy, paste and cut

### ISSUE THREAD

**Issue by bennyprasetia** [udos86/flutter-fast-forms#48]
Is there a special configuration that makes it work? (copy, paste, cut)

**Comment by Kaizodo** [comment]
Yup when selecting text it does not show cut , copy etc options tested on Nothing Phone 1, android 14
